website/members/models.py

```diff
@@ -25,6 +32,8 @@ class Member(models.Model):
 
     is_student = models.BooleanField()
 
+    student_type = models.CharField(max_length=3, choices=TYPE_STUDENT, blank=True)
```

Comment:
It should be mandatory for a member that checks the is_student checkbox to give a sudent_type